clients/java/revapi.json

```diff
@@ -34,6 +34,38 @@
           "code": "java.annotation.removed",
           "annotationType": "io.camunda.zeebe.client.api.ExperimentalApi",
           "justification": "The ExperimentalApi annotation is used to mark methods as 'in-development'. It is okay to remove it after a feature is implemented"
+        },
+        {
+          "ignore": true,
+          "code": "java.annotation.attributeValueChanged",
+          "annotationType": "io.camunda.zeebe.client.api.ExperimentalApi",
+          "justification": "The ExperimentalApi annotation is used to mark methods as 'in-development'. It is okay to change value field"
+        },
+        {
+          "ignore": true,
+          "code": "java.method.removed",
+          "old": "method io.camunda.zeebe.client.protocol.rest.UserTaskUpdateRequest io.camunda.zeebe.client.protocol.rest.UserTaskUpdateRequest::changeset(io.camunda.zeebe.client.protocol.rest.UserTaskUpdateRequestChangeset)",
+          "justification": "ADD YOUR EXPLANATION FOR THE NECESSITY OF THIS CHANGE"
+
+        },
+        {
+          "ignore": true,
+          "code": "java.method.returnTypeChanged",
+          "old": "method io.camunda.zeebe.client.protocol.rest.UserTaskUpdateRequestChangeset io.camunda.zeebe.client.protocol.rest.UserTaskUpdateRequest::getChangeset()",
+          "new": "method io.camunda.zeebe.client.protocol.rest.Changeset io.camunda.zeebe.client.protocol.rest.UserTaskUpdateRequest::getChangeset()",
+          "justification": "ADD YOUR EXPLANATION FOR THE NECESSITY OF THIS CHANGE"
+        },
+        {
+          "ignore": true,
+          "code": "java.method.removed",
+          "old": "method void io.camunda.zeebe.client.protocol.rest.UserTaskUpdateRequest::setChangeset(io.camunda.zeebe.client.protocol.rest.UserTaskUpdateRequestChangeset)",
+          "justification": "ADD YOUR EXPLANATION FOR THE NECESSITY OF THIS CHANGE"
+        },
+        {
+          "ignore": true,
+          "code": "java.class.removed",
+          "old": "class io.camunda.zeebe.client.protocol.rest.UserTaskUpdateRequestChangeset",
+          "justification": "ADD YOUR EXPLANATION FOR THE NECESSITY OF THIS CHANGE"
```

Comment:
@tmetzke I'm not sure about the justification, can you help me here ?

Reply:
The classes/methods are only internal and are not user-facing, thus it is OK that they change.